config/default.json

```diff
@@ -66,20 +67,33 @@
       },
       "jobs": {
         "new": {
-          "type": "Ingestion_New"
+          "type": "Ingestion_New",
+          "isUsedForPolling": true
```

Comment:
Aren't you prefer wrap the the job section by: "Polling" instead of duplicate a boolean value for each of the jobs?